docs/tips.md

```diff
@@ -63,11 +63,12 @@ to exiting input directly, without entering Vi mode, making it behave like a reg
 
 Save these lines as `~/.config/yazi/plugins/smart-enter.yazi/init.lua`:
 
+
 ```lua
 return {
 	entry = function()
 		local h = cx.active.current.hovered
-		ya.manager_emit(h and h.cha.is_dir and "enter" or "open", { hovered = true })
+		ya.manager_emit(h and h.cha.is_dir and "enter" or "open", { hovered = false })
```

Comment:
Could you keep it set to true, but just add a prompt to let users know they can switch it to false?